src/rTerminal.ts

```diff
@@ -71,6 +71,13 @@ export async function chooseTerminal(active: boolean = false) {
             if (rTermNameOpinions.includes(activeTerminalName)) {
```

Comment:
Is it `rTermNameOpinions` or `rTermNameOptions`? Looks like a typo introduced a while ago? Or I'm missing something?

Reply:
Definitely a typo. (Possibly by me.)